jwst/extract_1d/extract.py

```diff
@@ -2724,15 +2724,18 @@ def do_extract1d(input_model, ref_dict, smoothing_length=None,
                         input_model.meta.exposure.type)
 
     if (was_source_model or
-        isinstance(input_model, datamodels.MultiSlitModel) or
-        isinstance(input_model, datamodels.MultiProductModel)):
+        isinstance(input_model, datamodels.MultiSlitModel):
+#        isinstance(input_model, datamodels.MultiSlitModel) or
+#        isinstance(input_model, datamodels.MultiProductModel)):
 
         if was_source_model:            # from a SourceModelContainer?
             slits = [input_model]
-        elif isinstance(input_model, datamodels.MultiSlitModel):
+        else isinstance(input_model, datamodels.MultiSlitModel):
```

Comment:
This needs to remain `elif`, because it's got a clause following it. `else` can only be used "bare". This syntax error is causing all the CI tests to fail to build.